orttraining/orttraining/training_api/module.h

```diff
@@ -4,13 +4,27 @@
 #pragma once
 
 #include <string>
+#include <utility>
 #include "core/session/inference_session.h"
 #include "orttraining/training_api/utils.h"
 
 namespace onnxruntime {
 namespace training {
 namespace api {
 
+class EvalSessionWrapper : public InferenceSession {
```

Comment:
Can move this to the module.cc since it is only ever used there. Either in an unnamed namespace or in a locally defined class in the function.

Reply:
moved to locally defined in function